support/gathering-cluster-data.adoc

```diff
@@ -21,3 +21,22 @@ include::modules/support-gather-data.adoc[leveloffset=+1]
 
 // Obtain your cluster identifier
 include::modules/support-get-cluster-id.adoc[leveloffset=+1]
+
+// About sosreport
+include::modules/about-sosreport.adoc[leveloffset=+1]
+
+// Generating a sosreport
+include::modules/support-generating-a-sosreport-archive.adoc[leveloffset=+1]
+
+// Querying bootstrap node journal logs
+include::modules/querying-bootstrap-node-journal-logs.adoc[leveloffset=+1]
+
+// Querying cluster node journal logs
+include::modules/querying-cluster-node-journal-logs.adoc[leveloffset=+1]
+
+// Collecting a network trace from an OpenShift cluster node
+include::modules/support-collecting-network-trace.adoc[leveloffset=+1]
+
+// TODO: Extracting debug data from a cluster
+
+// TODO: Providing debug data to Red Hat Support
```

Comment:
Are we looking at how to use the red hat support tools? 
https://access.redhat.com/articles/445443

Reply:
Thanks Eric. I have now created a new module detailing three options to upload data to Red Hat, including details on how to use `redhat-support-tool` from within a toolbox container as part of an `oc debug` session. See `modules/support-providing-diagnostic-data-to-red-hat.adoc` for details.

There are also specific examples of steps to upload `tcpdump` capture files and `sosreport` archives to Red Hat for diagnosis, within `modules/support-collecting-network-trace.adoc` and `modules/support-generating-a-sosreport-archive.adoc` respectively.